TakashiAihara/pomodoro-next
# feature/button unit test

- Add unit test packages (vitest/testing-library/happy-dom)
- Add vitest config/Modify tsconfig/package.json for unit test
- Fix unit test failed bugs
- Summarize test reports into test-results
- Add unit test workflow, refactor test workflows
- change cache key for test
- Fix workflow 'if' property bugs
- Add storycap_deploy
- rename storycap_deploy
- modify on timing for test
- Revert "modify on timing for test"
- rename storycap_deploy
- Replace vrt jobs with docs/storycap branch
- add coverage check steps
- Fix tests workflow bugs, refactor
- Fix unit test workflow coverage path differ bug
- Add unplugin-auto-import and settings
- Modify vitest configs
- Add button unit test
